src/sentry/options/defaults.py

```diff
@@ -1421,7 +1421,7 @@
 )
 
 # The flag activates whether to send group attributes messages to kafka
-register("issues.group_attributes.send_kafka", default=False, flags=FLAG_MODIFIABLE_BOOL)
+register("issues.group_attributes.send_kafka", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
You can still keep both flags if you want as the options automator is still  being rolled out.

Reply:
Is there any need to besides having the option to use the _admin UI to update the option vs the repo?

Reply:
no. It is up to you. 
In the ideal world we would use only one channel, hopefully the automator. 
But we are not yet there, since the automator is still being fully rolled out, you may want to keep the admin as a backup.